lib/common/pyDriveUtil.py

```diff
@@ -59,13 +68,23 @@ def main():
                             help='Specify file name your want to update.', required=True)
     arg_parser.add_argument('-c', action='store', dest='input_content', default=None,
                             help='Specify content want to update.', required=True)
```

Comment:
maybe the `input_folder_uri`, `input_file_name`, and `input_content` can set as positional arguments.
so we can skip `-u` `-f` `-c` :P
ex:
```python
     arg_parser.add_argument('Folder_URI', default=None,
                             help='Specify the uri path of folder.', required=True)
     arg_parser.add_argument('Filename', default=None,
                              help='Specify file name your want to update.', required=True)
     arg_parser.add_argument('Content', default=None,
                              help='Specify content want to update.', required=True)
```